oozie-to-airflow/templates/mapreduce.tpl

```diff
@@ -24,10 +24,10 @@
     main_class=PARAMS['hadoop_main_class'],
     arguments=['{{ properties['mapreduce.input.fileinputformat.inputdir'] }}', '{{ properties['mapreduce.output.fileoutputformat.outputdir'] }}'],
     {% if hdfs_files %}
-    files=["{{ hdfs_files }}"],
+    files={{ hdfs_files | tojson }},
     {% endif %}
     {% if hdfs_archives %}
-    archives=["{{ hdfs_archives }}"],
+    archives={{ hdfs_archives | tojson }},
```

Comment:
These changes concern MapReduce, let's do them in a separate PR.

Reply:
A separate PR will have no reason to introduce this change. We agreed that every PR should be justified.

Reply:
Please explain what is the reason to change the mapreduce template in the "Add Spark Mapper" PR?